src/applications/applications.controller.ts

```
@@ -1,4 +1,4 @@
-import { Body, Controller, Post, Request, UseGuards, ConflictException, Get, Param, ParseIntPipe, Req } from '@nestjs/common';
+import { Body, Controller, Post, Request, UseGuards, ConflictException, Get, Param, ParseIntPipe, Req, Patch } from '@nestjs/common';
```

ESLint

[error] Replace `·Body,·Controller,·Post,·Request,·UseGuards,·ConflictException,·Get,·Param,·ParseIntPipe,·Req,·Patch·` with `⏎··Body,⏎··Controller,⏎··Post,⏎··Request,⏎··UseGuards,⏎··ConflictException,⏎··Get,⏎··Param,⏎··ParseIntPipe,⏎··Req,⏎··Patch,⏎`

(prettier/prettier)